### PR TITLE
Exposed upscaling_speed and idle_timeout_minutes to values.yaml, #25312

### LINKED ISSUE #25312: [Autoscaler] expose upscaling_speed  and idle_timeout_minutes to values.yaml when using Helm
### Description

The [upscaling_speed](https://docs.ray.io/en/latest/cluster/config.html#upscaling-speed)  and [idle_timeout_minutes](https://docs.ray.io/en/latest/cluster/config.html#cluster-configuration-idle-timeout-minutes) properties are useful and it already in [RayCluster](https://github.com/ray-project/ray/blob/master/deploy/charts/ray/templates/raycluster.yaml#L13-L15). However, it not expose to [values.yaml](https://github.com/ray-project/ray/blob/master/deploy/charts/ray/values.yaml).

I think exposing these two properties to `values.yaml` would be useful for the end-users. If this enhancement is meaningful, I could contribute to it.

### Use case

_No response_

### PR DESCRIPTION
**Pull request by orcahmlee**
Exposed upscaling_speed and idle_timeout_minutes to values.yaml, #25312
Signed-off-by: Andrew Li <orcahmlee@gmail.com>

## Related issue number

Closes #25312 

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: deploy/charts/ray/values.yaml [hunk 1]
```diff
@@ -6,6 +6,13 @@
 # It's recommended to build custom dependencies for your workload into this image,
 # taking one of the offical `rayproject/ray` images as base.
 image: rayproject/ray:latest
+# The autoscaler will scale up the cluster faster with higher upscaling speed.
+# E.g., if the task requires adding more nodes then autoscaler will gradually
```
Comment (DmitriGekhtman) [inline deploy/charts/ray/values.yaml:10]: A small grammar nit: remove "E.g.,"

**Comment by DmitriGekhtman** [comment]
Thank you for the PR!

It looks good to me.

Just one request: could you double check that `helm install` works without and without the newly added fields?
Everything looks correct to me, but we do not currently test the chart in the CI for each PR.
(Tests for this chart are run on a K8s cluster only during the release cycle.)

File: deploy/charts/ray/values.yaml (hunk 1 above)
Comment (orcahmlee) [reply deploy/charts/ray/values.yaml]: Done

**Comment by orcahmlee** [comment]
> Thank you for the PR!
> 
> It looks good to me.
> 
> Just one request: could you double check that `helm install` works without and without the newly added fields? Everything looks correct to me, but we do not currently test the chart in the CI for each PR. (Tests for this chart are run on a K8s cluster only during the process for each Ray release.)

Hi @DmitriGekhtman , thanks for your review.

I've already used the newly added fields on my GKE, and so far so good. If there has any test failure for Ray release, please tag me and I will fix it.

**Review by DmitriGekhtman — APPROVED** [review]
LGTM, many thanks!

Will let the CI run to completion before merging.

**Comment by DmitriGekhtman** [comment]
Test failures are unrelated.

**Pull request merged**